pkgs/misc/emulators/retroarch/cores.nix

```diff
@@ -674,7 +659,6 @@ in
 
   pcsx2 = mkLibRetroCore {
     core = "pcsx2";
-    version = "unstable-2021-11-27";
```

Comment:
No more version for this guy?

Reply:
It derivates the version from here: https://github.com/NixOS/nixpkgs/pull/149114/files#diff-86029dc4c3d1ca3e77939271748ead9d43e99e83df50a7d368337b59e64b3b30R58.

The only reason this version was set before because when I introduced this core in PR https://github.com/NixOS/nixpkgs/pull/147622 I didn't want to bump all other cores.

Reply:
Gotcha.